redirect_rules.py

```diff
@@ -79,12 +79,21 @@
     'oraclecloud'
 ]
 
+# URL Regex
+regex = re.compile(
+        r'^(?:http|ftp)s?://' # http:// or https://
```

Comment:
Include in the comment 'ftp:// or ftps://' for better awareness of what his regular expression is doing.

Reply:
Fixed in subsequent commit to remove ftp/ftps altogether as that doesn't really fit this redirection scenario